lakefile.lean

```diff
@@ -4,6 +4,7 @@ open Lake DSL
 
 def moreLeanArgs := #[
   "-DwarningAsError=true",
+  "-DsynthInstance.etaExperiment=true",
```

Comment:
Did you intend to commit this?

Reply:
oops! I was just experimenting with that locally now... :-) Will revert.